content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,135 @@
+
+# A Brief History of Decentralised Scaling
+
+This article will cover how the scaling solutions for decentralised systems in the Crypto world have evolved with regards to the consensus algorithms that have been used, what their problems are and how Flare tries to address them.
+We will cover how various blockchains fare in the realm of the [blockchain trilemma](https://vitalik.ca/general/2021/04/07/sharding.html) which roughly states that it is hard to achieve more than 2 ouf 3 from Security, Scalability and Decentralisation.
+
+The concept of Blockchain or Cryptography are not new and have been around before the 2000's.
+Still there was no concept of a truly digital currency/value due to the double spending problem.
+For example, say I have $100 worth of digital value and I send $100 to both Alice and Bob at the same time.
+There used to be no fully robust way to stop this from happening.
```

Comment:
I feel like this explanation of the problem is only clear to someone who already knows about it 😄 The way I usually see it described is more about how since it's digital value, nothing prevents you from "copying" it and sending it twice, or something along those lines, but I feel like if we provide an external link it might allow us not even to have to worry about explaining it :) WDYT?